package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "kenora-thistles-nuxt",
+  "version": "1.0.0",
```

Comment:
This would be the place to add `"license": "MIT"`, to make that clear, right?

Reply:
Added 👍